test/installer/test_apt_installer.py

```diff
@@ -0,0 +1,28 @@
+import sys
+from unittest import mock
+
+import unittest
+from mock import patch
+
+from colcon_bundle.installer.apt import AptBundleInstallerExtension
+
+
+class AptInstallerTests(unittest.TestCase):
+    @patch('colcon_bundle.installer.apt.get_ubuntu_distribution_version')
+    def test_should_load_not_on_ubuntu(self, get_version):
+        apt = AptBundleInstallerExtension()
+        get_version.side_effect = ValueError()
+        assert apt.should_load() == False
+
+    @patch('colcon_bundle.installer.apt.get_ubuntu_distribution_version')
+    def test_should_load_on_ubuntu(self, _):
+        sys.modules['apt'] = mock.MagicMock()
+        apt = AptBundleInstallerExtension()
+        assert apt.should_load() == True
+
+    @patch('colcon_bundle.installer.apt.get_ubuntu_distribution_version')
+    def test_should_load_on_ubuntu_no_python3_apt(self, _):
+        sys.modules['apt'] = None
```

Comment:
Should I be setting the value of this to a temporary value and then resetting it at the end of the test? Technically, they are unit tests so the `apt` module will never be used.